numerics/pid.hpp

```diff
@@ -0,0 +1,49 @@
+﻿
+#pragma once
+
+#include <deque>
+
+#include "quantities/named_quantities.hpp"
+#include "quantities/quantities.hpp"
+
+namespace principia {
+namespace numerics {
```

Comment:
Should this be in `physics`? It doesn’t feel like numerical analysis (there would be a bad argument for putting it in `integrators`, since it has an equally bad quadrature method in the middle of the implementation, but let’s not go there).

Reply:
Dunno, it doesn't seem very physical.  Think of it as applying a Laplace transform.